### DISCUSSION TITLE
User feedback: Using the Div... dropdown to create nested divs

### DISCUSSION THREAD

**Discussion by MayaGans** [quarto-dev/quarto-cli#484] [Q&A]
First of all I want to convey that quarto is SICK. When I clicked on the `Div..` option

<img width="257" alt="Screen Shot 2022-03-30 at 1 02 34 PM" src="https://user-images.githubusercontent.com/6053906/160921059-f91b9bfe-a109-4588-b445-523c040b3bf8.png">

I audibly gasped as it is so cool to see the class and ID in the little pill and be able to type content inside a div!!

<img width="561" alt="Screen Shot 2022-03-30 at 1 02 21 PM" src="https://user-images.githubusercontent.com/6053906/160921054-93f17d01-3bbe-4036-891f-6d0b8f47fc4c.png">

I just wanted to share some user feedback... my mental model was that once inside the div I could click from that dropdown again to insert a child component, but instead that just added a new class to the original div. 

<img width="564" alt="Screen Shot 2022-03-30 at 1 06 52 PM" src="https://user-images.githubusercontent.com/6053906/160921766-77a9b0f6-bdee-45d2-af84-5b216f4bffa2.png">

I ended up toggling over from `visual` to `source` and saw I could programmatically get the visual queue I was intending by writing

```
::: container
::: child_1
:::

::: child_2
:::
:::
```

![Uploading Screen Shot 2022-03-30 at 1.09.38 PM.png…]()

And I am PSYCHED!!!!!!!!!!!!!!

So cool seeing this is possible but figured I'd share my experience trying to use the Div.. dropdown. Again just a suggestion/user feedback since the functionality is all there --- this is tremendous!!

**Comment by jjallaire** [comment]
Glad you are enjoying the editor :-)

What you are seeing is a result of some unfortunate ambiguity in the editor: there are actually two "Div..." menus, one under Format and one under Insert. The Insert one will do what you are looking for.

Note also that you can use "Insert Anything" the `/` key as an alternative to the menu (https://quarto.org/docs/visual-editor/options.html#insert-anything)
